check_bazel_version.bzl

```diff
@@ -0,0 +1,39 @@
+def _parse_bazel_version(bazel_version):
+    # Remove commit from version.
+    version = bazel_version.split(" ", 1)[0]
+
+    # Split into (release, date) parts and only return the release
+    # as a tuple of integers.
+    parts = version.split("-", 1)
+
+    # Turn "release" into a tuple of strings
+    version_tuple = ()
+    for number in parts[0].split("."):
+        version_tuple += (int(number),)
+    return version_tuple
+
+# acceptable min_version <= version <= max_version
+def check_version():
+    check_bazel_version("0.5.3", "0.5.4")
```

Comment:
should in theory be 0.5.3 - 0.5.999 no ? (I mean if 0.5.5 was to come out it should work)